mkdocs/docs/contributing/newk8sstack.md

```diff
@@ -0,0 +1,125 @@
+---
+template: main.html
+---
+
+# Add a K8s Stack / Service Mesh / Ingress
+
+Performing Iter8 experiments requires RBAC rules, which are contained in [this Kustomize folder](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks) and are installed as part of Iter8 installation.
```

Comment:
"installed as part of Iter8 installation." -> "installed as part of the Iter8 installation."